test/index-test.js

```diff
@@ -4,15 +4,78 @@ const fs = require('fs')
 const jsdom = require('mocha-jsdom')
 const path = require('path')
 
-
 describe('index', () => {
-  
   jsdom({
+    html: fs.readFileSync(path.resolve(__dirname, '..', 'index.html'), 'utf-8'),
     src: fs.readFileSync(path.resolve(__dirname, '..', 'index.js'), 'utf-8')
   })
-  
 
-  it('runs', () => {
-    expect(true).to.be.true
+  describe('post form', () => {
+    it('exists and submits correctly', () => {
+      var postForm = document.getElementById("post-form");
+      expect(postForm).toExist("Must have a form with an id of post-form");
+      expect(postForm).toMatch(/onsubmit="createPost()/, "Form must submit to a createPost() function");
+    })
+  })
+
+  describe('functions', () => {
+    before(() => {
+      window._ = require('lodash')
```

Comment:
yeah no worries, I was able to piece it together from some of your earlier lab tests, which are very informative!